assets/js/components/notifications/AdBlockingRecoverySetupSuccessBannerNotification.test.js

```diff
@@ -48,10 +44,11 @@ import AdBlockingRecoverySetupSuccessBannerNotification from './AdBlockingRecove
 const mockTrackEvent = jest.spyOn( tracking, 'trackEvent' );
 mockTrackEvent.mockImplementation( () => Promise.resolve() );
 
-jest.mock( 'react-use' );
-
-mockUseIntersection.mockImplementation( () => ( {
-	isIntersecting: true,
+jest.mock( 'react-use', () => ( {
+	...jest.requireActual( 'react-use' ),
+	useIntersection: () => ( {
+		isIntersecting: true,
+	} ),
```

Comment:
Not sure how this wasn't broken before. The addition of `requireActual` here is something we do elsewhere and is necessary to preserve the normal function of other hooks from the package. Without this, the test was failing with `useMountedState is not a function`.